### PR TITLE
Fix update stage

### PR DESCRIPTION
**Pull request by soltysh**
This PR addressed following problems:

1. Removes comments from all staging godeps and makes sure that they are not placed there. This saves false positive during update when `Comment` field is cut by one digit.
2. Moves updating staging to a separate script (`hack/update-all-staging.sh`), `hack/update-all.sh` only runs verification of staging deps.  

@sttts ptal

### CODE REVIEW

File: hack/update-all.sh [hunk 1]
```diff
@@ -51,32 +51,38 @@ if ! $ALL ; then
...
 for t in $BASH_TARGETS
 do
-	echo -e "${color_yellow}Updating $t${color_norm}"
+	echo -e "${color_yellow}Running $t${color_norm}"
 	if $SILENT ; then
-		if ! bash "$KUBE_ROOT/hack/update-$t.sh" 1> /dev/null; then
-			echo -e "${color_red}Updating $t FAILED${color_norm}"
+		if ! bash "$KUBE_ROOT/hack/$t.sh" 1> /dev/null; then
+			echo -e "${color_red}Running $t FAILED${color_norm}"
+			if [[ $t == "verify*" ]]; then
```
Comment (sttts) [inline hack/update-all.sh]: doesn't the `*` have to be unquoted?

Comment (soltysh) [inline hack/update-all.sh]: Lemme double check...

File: hack/update-all.sh [hunk 2]
```diff
@@ -51,32 +51,38 @@ if ! $ALL ; then
...
 			if ! $ALL; then
 				exit 1
 			fi
 		fi
 	else
-		if ! bash "$KUBE_ROOT/hack/update-$t.sh"; then
-			echo -e "${color_red}Updating $t FAILED${color_norm}"
+		if ! bash "$KUBE_ROOT/hack/$t.sh"; then
+			echo -e "${color_red}Running $t FAILED${color_norm}"
+			if [[ $t == "verify*" ]]; then
```
Comment (sttts) [inline hack/update-all.sh]: dito

File: hack/update-all-staging.sh [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+if $SILENT ; then
+	echo "Running in the silent mode, run with -v if you want to see script logs."
+fi
+
+if ! $ALL ; then
+	echo "Running in short-circuit mode; run with -a to force all scripts to run."
+fi
+
+kube::util::ensure_clean_working_dir
+kube::util::ensure-temp-dir
```
Comment (sttts) [inline hack/update-all-staging.sh]: temp dir is implicit in ensure_godep_version. no need to have that here

File: hack/update-all.sh (hunk 1 above)
Comment (soltysh) [inline hack/update-all.sh]: Yup, fixing.

File: hack/update-all-staging.sh [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+kube::util::ensure_godep_version v74
+
+echo "Running godep restore"
```
Comment (sttts) [inline hack/update-all-staging.sh]: if you run the util.sh's godeps_restored before we can save quite some time if godeps are restored already.

**Comment by sttts** [comment]
Reviewed, ptal.

File: hack/update-all-staging.sh (hunk 4 above)
Comment (soltysh) [inline hack/update-all-staging.sh]: Makes sense, updating.

**Comment by soltysh** [comment]
@sttts updated, ptal

**Comment by sttts** [comment]
hack/update-all-staging.sh: line 53: kube::util::ensure_clean_working_dir: command not found

File: hack/update-all-staging.sh [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A single script that runs a predefined set of update-* scripts, as they often go together.
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${KUBE_ROOT}/cluster/lib/util.sh"
```
Comment (sttts) [inline hack/update-all-staging.sh]: s/cluster/hack/

Comment (soltysh) [inline hack/update-all-staging.sh]: aha, that was nasty copy&paste from `hack/update-all.sh` I'm guessing it needs update there as well.

Comment (soltysh) [inline hack/update-all-staging.sh]: Spoke too soon, that is for colors. I've added appropriate hack/lib/util.sh here as well.

**Comment by soltysh** [comment]
Updated and verified.

**Comment by sttts** [comment]
Verified.

/lgtm
/approve

**Comment by soltysh** [comment]
I'm setting milestone for 1.6, since this is fixing our tooling.

**Pull request merged**